apps/expo-go/android/expoview/src/main/java/host/exp/exponent/experience/ExperienceActivity.kt

```diff
@@ -227,14 +241,17 @@ open class ExperienceActivity : BaseExperienceActivity(), StartReactInstanceDele
     super.onWindowFocusChanged(hasFocus)
     // Check for manifest to avoid calling this when first loading an experience
     if (hasFocus && manifest != null) {
-      runOnUiThread { ExperienceActivityUtils.setNavigationBar(manifest!!, this@ExperienceActivity) }
+      lifecycleScope.launch {
+        ExperienceActivityUtils.setNavigationBar(
+          manifest!!,
+          this@ExperienceActivity
+        )
+      }
     }
   }
 
   private fun soLoaderInit() {
-    if (detachSdkVersion != null) {
-      SoLoader.init(this, false)
-    }
+    SoLoader.init(this, false)
```

Comment:
```suggestion
    SoLoader.init(this, OpenSourceMergedSoMapping)
```

there are so other `SoLoader.init()` inside expo-go, please try to update them all